fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -687,6 +688,41 @@ deleteStatement
     ;
 
 // ------------------------------------------- Routine Statement -----------------------------------------------------------
+createRoutineLoadStatement
+    : CREATE ROUTINE LOAD (db=qualifiedName '.')? name=identifier ON table=qualifiedName
+        (loadProperties (',' loadProperties)*)?
+        jobProperties?
+        FROM source=identifier
+        dataSourceProperties?
+    ;
+
+loadProperties
+    : (colSeparatorProperty)
+    | (rowDelimiterProperty)
+    | (COLUMNS columnProperties)
+    | (WHERE expression)
+    | (partitionNames)
+    ;
+
+colSeparatorProperty
+    : COLUMNS TERMINATED BY string
+    ;
+
+rowDelimiterProperty
+    : ROWS TERMINATED BY string
+    ;
+
+columnProperties
+    : '(' identifier (',' identifier)* (',' assignment)* ')'
```

Comment:
There are differences from the original old syntax

Reply:
[modify columnProperties](https://github.com/StarRocks/starrocks/pull/9981/commits/8c1c238d706edab9fa90b06544bee44763feb316)
fix this problem,thanks @HangyuanLiu